formidable/forms/__init__.py

```diff
@@ -55,6 +55,22 @@ def clean(self):
         return cleaned_data
 
 
+def get_dynamic_form_class_from_schema(schema, field_factory=None):
+    """
+    Schema Already contextualized
```

Comment:
I'm not a fan of this docstring. maybe it could be a bit more consistent, using a simple code sample, for example? And the first line should read:

> Return a dynamically generated and contextualized form class